release/release_workflow.wdl

```diff
@@ -235,151 +131,33 @@ task makeGithubRelease {
-        dependencyCommands = wdltoolDependencyCommands.updateCommand
-       }  
-       
-  call do_release as release_cromwell { input: 
-        organization = organization, 
-        repo = "cromwell", 
-        releaseV = cromwellPrep.version,
-        nextV = cromwellPrep.nextVersion,
-        updateVersionCommand = cromwellPrep.updateCommand,
-        dependencyCommands = cromwellDependencyCommands.updateCommand
-       }
-  
-  call wait_for_published_artifact as waitForLenthall { input: repo = "lenthall_2.12", version = release_lenthall.version }
-  call wait_for_published_artifact as waitForWdl4s { input: repo = "wdl4s-wdl_2.12", version = release_wdl4s.version }
-
-  call wait_for_published_artifact as waitForWdl4sCwl { input: repo = "wdl4s-cwl_2.12", version = release_wdl4s.version }
   
-  # Generates commands to update wdl4s dependencies
-  scatter(wdl4sDependency in wdl4sDependencies) {
-    String wdl4sDepName = wdl4sDependency.left
-    String wdl4sVersionName = wdl4sDependency.right
-    
-    call create_update_dependency_command as wdl4sDependencyCommands { input: 
-       dependencyName = wdl4sDepName,
-       newVersion = wdl4sVersionName,
-       dependencyFilePath = "build.sbt"
-    }
-  }
-  
-  # Generates commands to update wdltool dependencies
-  scatter(wdltoolDependency in wdltoolDependencies) {
-    String wdltoolDepName = wdltoolDependency.left
-    String wdltoolVersionName = wdltoolDependency.right
-    
-    call create_update_dependency_command as wdltoolDependencyCommands { input: 
-       dependencyName = wdltoolDepName,
-       newVersion = wdltoolVersionName,
-       dependencyFilePath = "build.sbt"
-    }
-  }
-  
-  # Generates commands to update cromwell dependencies
-  scatter(cromwellDependency in cromwellDependencies) {
-    String cromwellDepName = cromwellDependency.left
-    String cromwellVersionName = cromwellDependency.right
-    
-    call create_update_dependency_command as cromwellDependencyCommands { input: 
-       dependencyName = cromwellDepName,
-       newVersion = cromwellVersionName,
-       dependencyFilePath = "project/Dependencies.scala"
-    }
-  }
-  
-  File cromwellJar = release_cromwell.executionDir + "/target/scala-2.12/cromwell-" + cromwellPrep.version + ".jar"
-  # Version that was just released
-  Int cromwellVersionAsInt = cromwellPrep.version
+  File cromwellJar = do_release.executionDir + "/target/scala-2.12/cromwell-" + versionPrep.currentVersion + ".jar"
+  Int cromwellVersionAsInt = versionPrep.currentVersion
   # Previous version
-  Int cromwellPreviousVersion = cromwellVersionAsInt - 1
-  
+  Int cromwellPreviousVersionAsInt = cromwellVersionAsInt - 1
+
   call makeGithubRelease { input:
            githubToken = githubToken,
            organization = organization,
            cromwellJar = cromwellJar,
            newVersion = cromwellVersionAsInt,
-           oldVersion = cromwellPreviousVersion
+           oldVersion = cromwellPreviousVersionAsInt
```

Comment:
Just OOI, are we losing all separate versioning for subprojects? E.g. `wdl4s-0.15` is now just `cromwell-wdl-30`?

Reply:
I've been thinking lately that that wouldn't be a bad thing for book keeping purposes. The flip side is that if we don't touch the artist currently known as `wdl4s` that we're revving versions on people and that seems sucky.

Reply:
Yes all separate versioning was lost in the monorepoing, not in this PR.